Utils/src/Mt2Producer.cc

```diff
@@ -49,9 +49,9 @@ class Mt2Producer : public edm::global::EDProducer<> {
   edm::EDGetTokenT<reco::CandidateView> JetTok_;
   edm::EDGetTokenT<edm::View<pat::MET>> MetTok_;
 
-  vector<TLorentzVector> getHemispheres(vector<TLorentzVector> jets) const;
-  double getMT2Hemi(vector<TLorentzVector> jets, TLorentzVector metVec) const;
-  double computeMT2(TLorentzVector visaVec, TLorentzVector visbVec, TLorentzVector metVec) const;
+  vector<math::PtEtaPhiELorentzVector> getHemispheres(vector<math::PtEtaPhiELorentzVector> jets) const;
```

Comment:
this function is never actually defined or used, so just remove it

Reply:
Done